lib/PGcore.pm

```diff
@@ -527,7 +527,7 @@ sub store_persistent_data {  # will store strings only (so far)
 	my $self = shift;
 	my $label = shift;
 	my @content = @_;
-	$self->internal_debug_message("PGcore::store_persistent_data: storing $label in PERSISTENCE_HASH");
+	# $self->internal_debug_message("PGcore::store_persistent_data: storing $label in PERSISTENCE_HASH");
```

Comment:
I wouldn't be much of an industrial dev if I didn't call out that keeping around dead code in comments is generally an anti-pattern. I found a decent discussion on this here: https://blog.kentcdodds.com/please-don-t-commit-commented-out-code-53d0b5b26d5f

tl;dr: It confuses the next developer to come along, and doesn't really provide any context. All the context is kept around in revision control though, so deleting it is safe and git handles all the archiving.

Reply:
Thanks for the pointer to the blog.  It's quite informative.  
I'll remove the commented out code.  
Some of my habits were developed long before we were using git, or even CVS to version code.  :-)